packages/babel-plugin-transform-modules-commonjs/test/fixtures/misc/import-const-throw/output.js

```diff
@@ -45,33 +45,33 @@ _baz.Baz = (44, function () {
 } = ({}, function () {
   throw new Error('"' + "Baz" + '" is read-only.');
 }()));
-_foo.default = (_foo.default + 2, function () {
+_foo.default = _foo.default + (2, function () {
   throw new Error('"' + "Foo" + '" is read-only.');
 }());
```

Comment:
Can we place the read-only check before the assignment? Since the export is read-only, if we try catch the assignment, the export value should not be changed. Currently the `foo.default` is actually changed to `NaN` if the error is caught.